l1-contracts/contracts/bridge/ntv/NativeTokenVault.sol

```diff
@@ -333,6 +333,9 @@ abstract contract NativeTokenVault is INativeTokenVault, IAssetHandler, Ownable2
     /// @dev It does not perform any checks for the correctnesss of the token contract.
     /// @param _nativeToken The address of the token to be registered.
     function _unsafeRegisterNativeToken(address _nativeToken) internal {
+        if (assetId[_nativeToken] != bytes32(0)) {
```

Comment:
maybe better to throw an error? it will make it much more visible than silent ignore

Reply:
But why is it an error? 
It simply means this assetId was registered before